packages/design-system/src/components/web-components/ds-filter-chip/ds-filter-chip.tsx

```diff
@@ -0,0 +1,54 @@
+import { define } from '../preactement/define';
+import FilterChip from '../../FilterChip/FilterChip';
+import { FilterChipProps } from '../../FilterChip/FilterChip';
+import { parseBooleanAttr } from '../wrapperUtils';
+
+const attributes = [
+  'clear-label',
+  'class-name',
+  'label',
+  'root-id',
+  'size',
+  'use-alternate-icon',
+] as const;
+
+/* eslint-disable @typescript-eslint/no-namespace */
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      'ds-filter-chip': React.DetailedHTMLProps<
+        React.HTMLAttributes<HTMLElement> & {
+          [K in (typeof attributes)[number]]?: string;
+        },
+        HTMLElement
+      >;
+    }
+  }
+}
+/* eslint-enable */
+
+interface WrapperProps extends Omit<FilterChipProps, 'useAlternateIcon'> {
+  useAlternateIcon?: string;
+  clearLabel?: string;
+}
+
+const Wrapper = ({ clearLabel, label, useAlternateIcon, ...otherProps }: WrapperProps) => (
+  <FilterChip
+    ariaClearLabel={clearLabel ?? ''}
+    label={label}
```

Comment:
I noticed that the `label` prop is explicitly wired up here, but since it's just a string and doesn't seem to require any special handling, would it make sense to pass it through `otherProps` instead? I'm wondering if there's a specific reason for this approach.

Reply:
I think this was a typescript thing? The only required prop for FilterChip is a label, and the compiler wasn't picking up on whether it was present in the otherprops object or not, so I destructured it and explicitly passed it in to appease the typescript gods.